Tests/SwiftDocCTests/Rendering/DefaultAvailabilityTests.swift

```diff
@@ -374,4 +380,72 @@ class DefaultAvailabilityTests: XCTestCase {
         
         XCTAssertEqual(renderNode.deprecationSummary?.firstParagraph, expected)
     }
+    
+    func testUnconditionallyUnavailable() throws {
+        let infoPlist = """
+            <plist version="1.0">
+            <dict>
+                <key>CFBundleName</key>
+                <string>MyModule</string>
```

Comment:
minor: This property list key isn't used by DocC.

Reply:
Resolved here https://github.com/apple/swift-docc/pull/807/commits/c2a8b16dc10f2e9e624d9ebbb59964ad7b5a4db5